go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/riverqueue/riverui
 
-go 1.22
+go 1.22.5
```

Comment:
need to make tweaks to this based on what we've just learned, right?

Reply:
Changed it to 1.22. We have to keep this one on 1.22+ though because we're using Go 1.22's new HTTP routing features.